python/mdvtools/dbutils/mdv_server_app.py

```
@@ -19,10 +19,11 @@
 from psycogreen.gevent import patch_psycopg
 patch_psycopg()
 
-# Setup logging
-logger = logging.getLogger(__name__)
+# Setup logging using the centralized logging module
+from mdvtools.logging_config import get_logger
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)